index.html

```diff
@@ -12,24 +12,171 @@
+                    </ul>
+                    <button type="button" class="proj-link">
+                        See this Project &#x2192;
+                    </button>
+                </section>
+            </div>
+            
+            <div class="project4 disp">
+                <img src="img/project_4.jpg" alt="">
+                <section class="project4-details">
+                    <h1 class="project-name">
+                        Project name goes here
+                    </h1>
+                    <ul class="project-tecnologies">
+                        <li class="item1">HTML/CSS</li>
+                        <li class="item2">Ruby on Rails</li>
+                        <li class="item3">JavaScript</li>
+                    </ul>
+                    <button type="button" class="proj-link">
+                        See this Project &#x2192;
+                    </button>
+                </section>
+            </div>
+            
+            <div class="project5 disp">
+                <img src="img/project_5.jpg" alt="">
+                <section class="project5-details">
+                    <h1 class="project-name">
+                        Project name goes here
+                    </h1>
+                    <ul class="project-tecnologies">
+                        <li class="item1">HTML/CSS</li>
+                        <li class="item2">Ruby on Rails</li>
+                        <li class="item3">JavaScript</li>
+                    </ul>
+                    <button type="button" class="proj-link">
+                        See this Project &#x2192;
+                    </button>
+                </section>
+            </div>
+            
+            <div class="project6 disp">
+                <img src="img/project_6.jpg" alt="">
+                <section class="project6-details">
+                    <h1 class="project-name">
+                        Project name goes here
+                    </h1>
+                    <ul class="project-tecnologies">
+                        <li class="item1">HTML/CSS</li>
+                        <li class="item2">Ruby on Rails</li>
+                        <li class="item3">JavaScript</li>
+                    </ul>
+                    <button type="button" class="proj-link">
+                        See this Project &#x2192;
+                    </button>
+                </section>
+            </div>
+        </section>
+        <section class="about">
+
+            <div class="about-me disp">
+                <h1 class="about-head">About</h1>
+                <p class="about-text">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna 
+                    aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+                </p>
+                <img src="img/get_cv.svg" alt="" class="get-cv">
```

Comment:
- [ ] Please add a button instead of an image and add interactions for it (hover, active)